common/djangoapps/course_modes/signals.py

```diff
@@ -35,3 +46,47 @@ def _listen_for_course_publish(sender, course_key, **kwargs):  # pylint: disable
 def _should_update_date(verified_mode):
     """ Returns whether or not the verified mode should be updated. """
     return not(verified_mode is None or verified_mode.expiration_datetime_is_explicit)
+
+
+def update_masters_access(item, verified_id, masters_id):
+    """
+    Update the XBlock's group access to allow the master's group,
+    in addition to the verified content group.
+    """
+    group_access = item.group_access
+    enrollment_groups = group_access.get(ENROLLMENT_TRACK_PARTITION_ID, None)
+    if enrollment_groups is not None:
+        if verified_id in enrollment_groups and masters_id not in enrollment_groups:
+            enrollment_groups.append(masters_id)
+            item.group_access = group_access
+            return True
```

Comment:
I feel that this should return False instead of relying on the implicit None return and that None being falsy. I feel that it's better to be explicit. If this function returns True when it successfully updates the master's access, then I think it should return False when it doesn't. What do you think?